packages-exp/auth-exp/test/integration/flows/custom.local.test.ts

```diff
@@ -146,6 +150,62 @@ describe('Integration test: custom auth', () => {
     expect(customUser.uid).not.to.eql(anonUser.uid);
   });
 
+  context('in passthrough mode', () => {
+    beforeEach(async () => {
+      const updatedConfig = await updateEmulatorProjectConfig(
+        JSON.stringify({
+          usageMode: 'PASSTHROUGH'
+        })
+      );
+      expect(updatedConfig).to.eql({
+        signIn: { allowDuplicateEmails: false },
+        usageMode: 'PASSTHROUGH'
+      });
+    });
+
+    afterEach(async () => {
+      await updateEmulatorProjectConfig(
+        JSON.stringify({
+          usageMode: 'DEFAULT'
+        })
+      );
+    });
+
+    xit('signs in with custom token in passthrough mode', async () => {
+      const cred = await signInWithCustomToken(auth, customToken);
+      expect(auth.currentUser).to.eq(cred.user);
+      expect(cred.operationType).to.eq(OperationType.SIGN_IN);
+
+      const { user } = cred;
+      expect(user.isAnonymous).to.be.false;
+      expect(user.uid).to.eq(uid);
+      expect((await user.getIdTokenResult(false)).claims.customClaim).to.eq(
+        'some-claim'
+      );
+      expect(user.providerId).to.eq('firebase');
+      const additionalUserInfo = await getAdditionalUserInfo(cred)!;
+      expect(additionalUserInfo.providerId).to.be.null;
+      expect(additionalUserInfo.isNewUser).to.be.false;
+    });
+
+    xit('token can be refreshed in passthrough mode', async () => {
+      setCustomTokenProvider(auth, {
+        async getCustomToken(): Promise<string> {
+          return JSON.stringify({
+            uid,
+            claims: {
+              customClaim: 'some-claim'
```

Comment:
Could you change this claim here to some other string and then assert it gets updated after the refresh (to ensure this code gets called)

Reply:
Done!